src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -9,8 +9,8 @@
 NODE_EXACT_VERSION_DEFAULT = "10.14.1"
 NETCORE_VERSION_DEFAULT = "3.1"
 ASPDOTNET_VERSION_DEFAULT = "4.8"
-DOTNET_VERSION_DEFAULT = "5.0"
-DOTNET_TARGET_FRAMEWORK_STRING = "net5.0"
```

Comment:
I don't think we should be removing .NET5 this is not out of support

Reply:
I think for now default should continue to be .NET5 since it's GA

.NET6 is in preview, we would do this switch once it goes GA in nov.

Reply:
gotcha, changed back